addons/lunch/wizard/lunch_confirm_view.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.ui.view" id="view_lunch_confirm_order_form">
+        <field name="name">lunch.confirm.order.form</field>
+        <field name="model">lunch.confirm.order</field>
+        <field name="arch" type="xml">
+            <form string="Confirm meals">
+                <separator string="Did you receive these meals?"/>
+                <p class="oe_grey">
+                    Once a meal is received (confirmed) a new cash move is created for the employee.
+                </p>
+                <footer>
+                    <button name="confirm" string="Receive Meals" type="object" class="oe_highlight"/>
+                    or
+                    <button name="cancel" string="Cancel" class="oe_link" special="cancel" />
+                </footer>
+            </form>
+        </field>
+    </record>
+
+    <act_window id="confirm_order_lines"
```

Comment:
use the `<record>` notation, or align all the attribute for the code to be easier to read ;)